.github/workflows/update-operators-cache.yml

```
@@ -0,0 +1,56 @@
+name: Update Aggregator Cache Info
+
+on:
+  workflow_dispatch:
+    inputs:
+      mainnet-blob-id:
+        description: 'Mainnet blob ID'
+        required: true
+        type: string
+        default: 'bfH-ZqYV5faXTiq2-FVBT2cLQD61VYRuDje0cIkMWMo'
+      testnet-blob-id:
+        description: 'Testnet blob ID'
+        required: true
+        type: string
+        default: 'bfH-ZqYV5faXTiq2-FVBT2cLQD61VYRuDje0cIkMWMo'
+  schedule:
+    - cron: '0 11 * * 1'  # Every Monday at 11:00 UTC
+
+env:
+  MAINNET_BLOB_ID: ${{ github.event.inputs.mainnet-blob-id || 'bfH-ZqYV5faXTiq2-FVBT2cLQD61VYRuDje0cIkMWMo' }}
+  TESTNET_BLOB_ID: ${{ github.event.inputs.testnet-blob-id || 'bfH-ZqYV5faXTiq2-FVBT2cLQD61VYRuDje0cIkMWMo' }}
+
+jobs:
+  update-cache:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: cd scripts/cache-inference && npm install
```

Nit: The following is a bit cleaner:
```suggestion
        working-directory: ./scripts/cache-inference
        run: npm install
```